### ISSUE TITLE
Handle absolute paths on Windows

### ISSUE THREAD

**Issue by vojtajina** [karma-runner/karma#251]
Based on https://github.com/vojtajina/testacular/issues/249

Let's add another `if (windows)` and handle the absolute paths on windows to something like `/absolute/c/path/to/file.js`.

**Comment by willprice** [comment]
Is there any workaround for this at the moment as it is preventing me from using Karma as my test runner.

**Comment by Dignifiedquire** [comment]
Not that I know about sorry, but I hope to get this fixed soon

**Comment by willprice** [comment]
That's good to know, thanks for getting back to me so quickly.

**Comment by n370** [comment]
Hey @dignifiedquire is there something I can do to help getting this corrected for the next release?

**Comment by igomezal** [comment]
With karma v4.0.0 gives me the following error: `[web-server]: 404: /absoluteC:/Users/...`, I am not sure but is it possible that it is not creating the correct route?

Is there any workaround to make it work on windows?

**Comment by johnjbarton** [comment]
Every commit is tested on Windows, eg https://ci.appveyor.com/project/dignifiedquire/karma/builds/22080177/job/oq6xvy45jl3fg0dx

So whatever problem you are having, it is not fundamental to karma.

**Comment by igomezal** [comment]
Yeah, you are right, it could be a problem with the latest version of karma-webpack (alpha) because the previous one seems to work fine. (https://github.com/webpack-contrib/karma-webpack/issues/396)

I would keep looking for the problem.
